src/components/main/TeamMembersList.jsx

```diff
@@ -40,7 +41,7 @@ TeamMembersList.propTypes = {
     React.PropTypes.shape({
       image_url: React.PropTypes.string,
       name: React.PropTypes.string.isRequired,
-      job_title: React.PropTypes.string.isRequired,
+      job_title: React.PropTypes.string,
```

Comment:
If it's not required, shouldn't you be handling the above `||` logic with a PropTypes default value? For that matter, shouldn't this be failing linting since it has a non-required Prop without a default value?

Reply:
I was wanting to do this! But I don't think you can set defaults for nested proptypes, actually haven't googled it, but I'll check now

Reply:
https://github.com/facebook/react/issues/2568

Seems it won't happen. In this case I'm not sure if we should just disable the eslint rule about defaultProps and handle all defaults ourselves like we did before? I quite liked the linter enforcing defaults though

Reply:
I mean, this doesn't seem to actually be failing linting so it's fine. If it's not possible, it's not possible 😒

Reply:
Yeah :(